doc/api/cli.md

```diff
@@ -193,6 +193,14 @@ added: v11.8.0
 
 Enable experimental diagnostic report feature.
 
+### `--experimental-self-specifier`
```

Comment:
The naming on this flag feels weird. Perhaps just `--experimental-require-self`?

Reply:
Yeah, not a huge fan of the current name myself. Happy to call it `require-self` or `import-self`  (`load-self`?). Will default to `require-self` if nobody else speaks up.

Reply:
It applies to both CommonJS and ESM, so require or import seem poor fits. How about `resolve-self`?

Reply:
Works for me, `resolve-self`.